pkg/vsphere/simulator/folder.go

```diff
@@ -145,6 +145,35 @@ func (f *Folder) CreateFolder(c *types.CreateFolder) soap.HasFault {
 	return r
 }
 
+type StoragePod struct {
```

Comment:
Might be helpful to have a comment for StoragePod here? you mentioned it is storage cluster in a test method. 😄